### PR TITLE
feat: now the expected folderPath parameter is an id

### PR DESCRIPTION
**Pull request by doubleface**
This is an update which must be done in all the existing folder, if you expect that your connector saves its files in the good folder.

### CODE REVIEW

**Comment by y-lohse** [comment]
Didn't it work with a folder name? I was under the impression that it did.

**Comment by doubleface** [comment]
Yes it worked with a folder name before, but the stack run it passing a folder id to avoid giving unexpected folder paths.

**Comment by doubleface** [comment]
ping @y-lohse ^

File: README.md [hunk 1]
```diff
@@ -75,7 +77,7 @@ The connector is run by calling npm start with the following envrionment variabl
     "attributes":{
       "arguments":{
         "account":"cf31eaef5d899404a7e8c3737c1c2d1f",
-        "folder_to_save":"folderPath",
+        "folder_to_save":"folderPathId",
```
Comment (gregorylegarec) [inline README.md]: Isn't `folderId` a property name ?

Comment (doubleface) [inline README.md]: No, this will be a real folder ID. This is just a placeholder in this example

**Pull request merged**